frontend/src/metabase/meta/Card.spec.js

```diff
@@ -59,7 +59,7 @@ describe("metabase/meta/Card", () => {
                 expect(parseUrl(url)).toEqual({
                     pathname: "/question",
                     query: {},
-                    card: card,
+                    card: dissoc(card, "id")
```

Comment:
Basically reverting the test to the previous version. Now that we have `original_card_id` we don't have to pass the `id` property around.

I should probably include `original_card_id` some of those test, though.